EssentialApp/EssentialApp/SceneDelegate.swift

```diff
@@ -56,7 +56,10 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 	}
 
 	private func showComments(for image: FeedImage) {
-		fatalError("Must be implemented")
+		let detailsViewController = ImageCommentsUIComposer.imageCommentsComposedWith(
+			imageCommentsLoader: makeRemoteImageCommentsLoader(imageID: image.id.uuidString)
+		)
+		(window?.rootViewController as? UINavigationController)?.pushViewController(detailsViewController, animated: true)
```

Comment:
You shouldn't extract the navigation from the window. There's a property for it:

```suggestion
		navigationController.pushViewController(detailsViewController, animated: true)
```